packages/nextra-theme-docs/package.json

```diff
@@ -61,7 +61,6 @@
   "devDependencies": {
     "@types/flexsearch": "^0.7.2",
     "@types/react": "^17.0.39",
-    "autoprefixer": "^10.2.6",
```

Comment:
don't need in docs since `autoprefixer` is already installed in the root `package.json`